### PR TITLE
Enable ppc64le and s390x in CSV

### PR DESCRIPTION
**Pull request by krishvoor**
Enables ppc64le and s390x in CSV
Signed-off-by: Krishna Harsha Voora <krishvoor@in.ibm.com>

### CODE REVIEW

**Review by eguzki — DISMISSED** [review]

**Comment by eguzki** [comment]
you need to run `make bundle`

File: bundle/manifests/apicast-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -28,6 +28,10 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
     repository: https://github.com/3scale/apicast-operator
     support: Red Hat
+  labels:
```
Comment (eguzki) [inline bundle/manifests/apicast-operator.clusterserviceversion.yaml:31]: changes needs to be done in the config/manifests/bases/apicast-operator.clusterserviceversion.yaml

Comment (eguzki) [reply bundle/manifests/apicast-operator.clusterserviceversion.yaml:31]: then run `make bundle`

Comment (krishvoor) [reply bundle/manifests/apicast-operator.clusterserviceversion.yaml:31]: this is done, resolving this conversation

Comment (eguzki) [reply bundle/manifests/apicast-operator.clusterserviceversion.yaml:31]: I do not see any change

Comment (krishvoor) [reply bundle/manifests/apicast-operator.clusterserviceversion.yaml:31]: The changes should be visible now :)

**Review by eguzki — APPROVED** [review]

**Pull request merged**